cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/AwsClusterSettingsStep.tsx

```diff
@@ -105,21 +105,21 @@ function AwsClusterSettingsStep(props: Partial<StepProps>) {
     };
 
     const osImages = (getResource(AWS_FIELDS.OS_IMAGE, awsState) || []) as AWSVirtualMachine[];
-    let initialSelectedInstanceTypeId = awsState[STORE_SECTION_FORM].NODE_PROFILE;
+    let initialSelectedInstanceTypeId = awsState[STORE_SECTION_FORM][AWS_FIELDS.NODE_PROFILE];
 
-    if (awsState[STORE_SECTION_FORM].OS_IMAGE) {
-        setValue(AWS_FIELDS.OS_IMAGE, awsState[STORE_SECTION_FORM].OS_IMAGE.name);
+    if (awsState[STORE_SECTION_FORM][AWS_FIELDS.OS_IMAGE]) {
+        setValue(AWS_FIELDS.OS_IMAGE, awsState[STORE_SECTION_FORM][AWS_FIELDS.OS_IMAGE].name);
     }
     if (!initialSelectedInstanceTypeId) {
         initialSelectedInstanceTypeId = nodeInstanceTypes[0].id;
         setValue(AWS_FIELDS.NODE_PROFILE, initialSelectedInstanceTypeId);
         onFieldChange(AWS_FIELDS.NODE_PROFILE, initialSelectedInstanceTypeId);
     }
 
-    const [selectedInstanceTypeId, setSelectedInstanceTypeId] = useState(initialSelectedInstanceTypeId);
+    // const [selectedInstanceTypeId, setSelectedInstanceTypeId] = useState(initialSelectedInstanceTypeId);
```

Comment:
nit: any reason to leave these two commented lines in the code?

Reply:
nice catch. fixed.